.github/workflows/documentation.yml

```diff
@@ -23,52 +20,20 @@ jobs:
       - name: Install dependencies
         run: |
             sudo apt update
-            sudo apt install openmpi-bin libopenmpi-dev ccache casacore-dev doxygen graphviz
-            if [[ "$USE_CONAN" = 1 ]]; then
-              pip install conan
-            else
-              sudo apt install libeigen3-dev libspdlog-dev libtiff-dev libcfitsio-dev
-              sudo apt install libbenchmark-dev libboost-all-dev libyaml-cpp-dev
-              git clone https://github.com/catchorg/Catch2.git -b v3.4.0
-              mkdir Catch2/build
-              cd Catch2/build
-              cmake .. -DBUILD_TESTING=OFF -DCMAKE_INSTALL_PREFIX=${{github.workspace}}/build
-              make -j$(nproc --ignore 1) install
-              cd -
-              wget --no-check-certificate --no-verbose http://www.fftw.org/fftw-3.3.10.tar.gz -O- | tar --no-same-owner -xz;
-              cd fftw-3.3.10
-              ./configure --prefix=${{github.workspace}}/build --enable-shared
-              make -j$(nproc --ignore 1) install CFLAGS=-fPIC
-              # Fix bug in FFT3 (cf. https://github.com/FFTW/fftw3/issues/332)
-              sed -i -e 's/^.*FFTW3LibraryDepends.cmake.*$//1' ${{github.workspace}}/build/lib*/cmake/*/FFTW3Config.cmake
-              cd -
-            fi
-
-      - name: Install Tensorflow API on Ubuntu
-        uses: UCL/install-tensorflow-action@main
-        with:
-          version: 2.11.0
-          os: linux
-
-      - name: Checkout cppflow repo
-        uses: actions/checkout@v3
-        with:
-          repository: UCL/cppflow.git
-          path: cppflow
-          ref: master
-
-      - name: Create cppflow package
-        run: |
-          if [[ "USE_CONAN" = 1 ]]; then
-            conan detect profile
-            conan create ./cppflow -s compiler.cppstd=17
-          else
-            mkdir cppflow/build
-            cd cppflow/build
-            cmake .. -DCMAKE_INSTALL_PREFIX=${PWD}
-            #cmake .. -DCMAKE_INSTALL_PREFIX=${{github.workspace}}/build
+            sudo apt install openmpi-bin libopenmpi-dev ccache casacore-dev doxygen graphviz libeigen3-dev libspdlog-dev libtiff-dev libcfitsio-dev libbenchmark-dev libboost-all-dev libyaml-cpp-dev
+            git clone https://github.com/catchorg/Catch2.git -b v3.4.0
+            mkdir Catch2/build
+            cd Catch2/build
+            cmake .. -DBUILD_TESTING=OFF -DCMAKE_INSTALL_PREFIX=${{github.workspace}}/build
```

Comment:
Can we build with tests off so that we don't need Catch2 here?

Reply:
Yes sir! 🫡